### PR TITLE
buildin passwordmanager + plugin options

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: lib/passport/strategy.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
 var
   q = require('q'),
+  PasswordManager = require('../passwordManager'),
```
Comment (ballinette) [inline lib/passport/strategy.js]: require not needed, the paswordManager is injected by the context.

File: lib/passwordManager.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var
+  PasswordManager,
```
Comment (ballinette) [inline lib/passwordManager.js]: variable not used

**Comment by ballinette** [comment]
+1, but waiting for removing passwordManager from Kuzzle core before merging ;)

**Comment by jenow** [comment]
+1

File: package.json [hunk 3]
```diff
@@ -3,10 +3,6 @@
   "version": "0.0.1",
   "description": "Kuzzle plugin to log-in users",
   "main": "./lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kuzzleio/kuzzle-plugin-auth-passport-local.git"
-  },
   "keywords": [
```
Comment (j33f) [inline package.json]: no more git repo ? why ?

File: package.json [hunk 4]
```diff
@@ -21,6 +17,9 @@
   },
   "pluginInfo": {
     "defaultConfig": {
+      "secret": "jbsB8r69PFP39KdLtjVr25Z22",
+      "algorithm": "sha1",
+      "digest": "hex"
```
Comment (j33f) [inline package.json]: not that secret

**Comment by scottinet** [comment]
@ballinette > no need. I deployed this plugin in NPM and we updated Kuzzle's master branch to use the `1.0.0` version of this plugin. This PR will tag this plugin as version `1.1.0` so there shouldn't be any conflict.

**Pull request merged**